### PR TITLE
Add example of removing a DataDep for the docs

### PR DESCRIPTION
**Pull request by briochemc**

### CODE REVIEW

**Comment by briochemc** [comment]
Old PR with some doc addition you asked for :) + a PR to add the CITATION.bib file

File: docs/src/z20-for-pkg-devs.md [hunk 1]
```diff
@@ -191,6 +191,14 @@ It attempts to use webscraping and such to workout what should be in the registr
...
 (E.g. remove excessive information in the message)
 
+### Removing data
+
+Sometimes you don't need the data anymore.
+You can remove files from within Julia using the `rm` command.
+If you had registered a `DataDep` called `MyDataName`, then you can remove it with
+```
+rm(datadep"MyDataName"; recursive=true)
+```
```
Comment (oxinabox) [inline docs/src/z20-for-pkg-devs.md]: Cool thanks.
I think this is probably better in the docs for end-users.

https://github.com/oxinabox/DataDeps.jl/blob/master/docs/src/z10-for-end-users.md

What do you think?

**Comment by briochemc** [comment]
Yes of course! Moved it there?

**Comment by oxinabox** [comment]
Thanks!

**Pull request merged**